Master-Game/Assets/Maps/Jay01.json

```diff
@@ -0,0 +1,104 @@
+﻿{
+  "name": "Jay01",
+  "version": "1.0",
+  "rows": 4,
+  "cols": 4,
+  "Tiles": [
+    {
+      "type": 1,
+      "xCoord": 0,
```

Comment:
this can still be call x and y

Reply:
According to @wilsont3 , I think they need to be the same name.

Reply:
When I tested it, it does not look like the names needed to be the same. But maybe we should keep them the same for clarity?

Reply:
sure